app/Http/Controllers/UserController.php

```diff
@@ -170,6 +170,9 @@ public static function statusesForUser(User $user): ?LengthAwarePaginator {
                     })->orderByDesc('created_at')->paginate(15);
     }
 
+    /**
+     * @todo remove twitterUrl after implemented ID-Link in vue Template
```

Comment:
See https://github.com/Traewelling/traewelling/pull/467#issuecomment-886096709

Reply:
Don't you think the method would be unnecessary if it just returned a string without doing anything?

Reply:
Hm. Maybe.